ToolBox/GoogleDownloader/GoogleDataDownloader.cs

```diff
@@ -27,13 +29,6 @@ namespace QuantConnect.ToolBox.GoogleDownloader
     /// </summary>
     public class GoogleDataDownloader : IDataDownloader
     {
-        // q = SYMBOL
-        // i = resolution in seconds
-        // p = period in days
-        // ts = start time
-        // Strangely Google forces CHLO format instead of normal OHLC.
-        private const string UrlPrototype = @"http://www.google.com/finance/getprices?q={0}&i={1}&p={2}d&f=d,c,h,l,o,v&ts={3}";
-
```

Comment:
Looks like we just moved this somewhere else in the file, obfuscating the diff a little bit.